### DISCUSSION TITLE
spinner fade in editor

### DISCUSSION THREAD

**Discussion by TrollMrRicky** [ppy/osu#22949] [Ideas] [answered]
Just trying out lazer editor again

Trying to place and move objects after a spinner is really annoying as the spinner's fade lasts a really long time - meaning if I'm attempting to select an object, I select the spinner instead

 ![image](https://user-images.githubusercontent.com/65743669/227747733-87cd8374-3cfa-4932-9593-4a0444d93464.png)

Here I was clicking on the slider, but it selects the spinner although it was finished a whole 1/1 beat before

![image](https://user-images.githubusercontent.com/65743669/227747783-379a5a18-43c7-4535-bfbc-5a992b7e0bbf.png)

Not sure if it's just me

Thanks

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23037
